### PR TITLE
Update Qt on Mac to 5.12.10, for better compatibility with Big Sur

### PR DESCRIPTION
**Pull request by softins**
Note that this makes the earliest version of macOS supported by the release builds
to be 10.12 (Sierra), and drops support for Yosemite and El Capitan.

### CODE REVIEW

**Review by hoffie — APPROVED** [review]

**Comment by softins** [comment]
Waiting for feedback from someone on Facebook before I merge this.

**Comment by hoffie** [comment]
> Waiting for feedback from someone on Facebook before I merge this.

If the feedback is positive, we should link this PR to #1628.

**Review by pljones — APPROVED** [review]

**Comment by JohannesBrx** [comment]
> drops support for Yosemite and El Capitan

Unfortunately, I cannot install the official MacOS builds of Jamulus any more. At least I can still build Jamulus locally. 
(And yes, I know it's getting time for a new OSX version...)

**Comment by hoffie** [comment]
@JohannesBrx Any specific reason why you require the old version?

**Comment by JohannesBrx** [comment]
> Any specific reason why you require the old version?

Compatibility with legacy software... I assume that it will be broken with a newer MacOS version.

**Comment by hoffie** [comment]
Understood.
If there is still comprehensible demand for this support, we could think about providing an additional Mac build for old Mac versions. This should be possible via some updates/refactoring of the autobuild scripts. I think we should only do this if there is enough demand, though. As far as I understand, those Mac vesions are unsupported by Apple. Using such an old Qt build for Jamulus may also keep known (Qt) security issues unfixed, so it needs proper considerations.

**Comment by softins** [comment]
> > drops support for Yosemite and El Capitan
> 
> Unfortunately, I cannot install the official MacOS builds of Jamulus any more.

Interested to know what happens when you try. Does it just refuse to install? Or install and fail to run?

Which version of macOS are you running?

**Comment by JohannesBrx** [comment]
> Interested to know what happens when you try. Does it just refuse to install? Or install and fail to run?
>
> Which version of macOS are you running?

I can still open the .dmg file and confirm the software license. But when I want to run Jamulus, I get the following error message, which stops execution after confirmation. As shown in the screenshot, I am using MacOS 10.11. I think I will try upgrading to High Sierra and see what happens to my legacy software...

image

**Comment by ann0see** [comment]
@JohannesBrx I'm not sure if you know, but now we provide two builds of Jamulus (to also support older versions of macOS)

**Pull request merged**
